src/js/me-shim.js

```diff
@@ -868,12 +869,21 @@ mejs.YouTubeApi = {
 					var originalSeekTo = settings.pluginMediaElement.pluginApi.seekTo.bind(player);
 
 					settings.pluginMediaElement.pluginApi.seekTo = function(seconds, allowSeekAhead) {
+						var startingTime = player.getCurrentTime();
+
+						var fireEventsAfterSeek = function() {
+							clearTimeout(seekTimeoutId);
+							seekTimeoutId = setTimeout(function() {
+								if (player.getCurrentTime() === startingTime) {
```

Comment:
I'd just add a comment saying that a seek has occurred if the player time has changed.

Also, just checking, if the player is playing and we seek, could the player continue playing and break this? should we check if the current time is >= the seconds? Just a thought?

Reply:
Regarding the scenario where the player is in the playing state and we seek:
My observation using Slow 3G in the Chrome dev tools and this change is that the playback stops immediately when a seek occurs, the spinner spins while it buffers, and then it seeks to exactly where we expect, even with multiple clicks (seeks) on the timebar. 

As far as checking for `>=`, we can always seek backward to an unbuffered part of a video, so I wouldn't want to make that assumption.

Reply:
duh. Okay.